srcpkgs/tuxguitar/template

```diff
@@ -0,0 +1,63 @@
+# Template file for 'tuxguitar'
+pkgname=tuxguitar
+version=1.5.1
+revision=1
+wrksrc="${pkgname}-${version}-src"
+hostmakedepends="apache-maven"
+makedepends="alsa-lib-devel fluidsynth-devel jack-devel"
+depends="virtual?java-runtime gtk+3-devel hicolor-icon-theme
+ desktop-file-utils"
+short_desc="Multitrack guitar tablature editor and player"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
```

Comment:
...

Reply:
Mh... I just had copied you template. Can set the Maintainer to myself.

Reply:
It doesn't matter